src/main/java/build/buf/protovalidate/exceptions/CompilationException.java

```diff
@@ -14,12 +14,13 @@
 
 package build.buf.protovalidate.exceptions;
 
-/**
- * {@link CompilationException} extends {@link ValidationException} is returned when a constraint
- * fails to compile. This is a fatal error.
- */
+/** CompilationException is returned when a constraint fails to compile. This is a fatal error. */
 public class CompilationException extends ValidationException {
   public CompilationException(String message) {
     super(message);
   }
+
+  public CompilationException(String message, Throwable cause) {
```

Comment:
Added overridden constructor so if we have an Exception cause, we can preserve it for debugging.